### PR TITLE
Global market

### PR DESCRIPTION
**Pull request by maheshmuttinti**

### CODE REVIEW

File: database/migrations/20220322155856-create-gm-products-table.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        allowNull: false,
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4,
+      },
+      name: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      category: {
```
Comment (rajeshn95) [inline database/migrations/20220322155856-create-gm-products-table.js:22]: @maheshmuttinti why do you need this `category` column?

File: app/functions/getGMBidsSearchFilterQueries.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    */
+  let searchItem = query.search;
+
+  console.log("gm-products searchItems:", searchItem);
+
+  const productNames = query?.product_names
+    ?.split(",")
+    ?.map((productName) => productName);
+  const brands = query?.brands?.split(",")?.map((brand) => brand);
+  const cities = query?.cities?.split(",")?.map((city) => city);
```
Comment (rajeshn95) [inline app/functions/getGMBidsSearchFilterQueries.js:16]: @maheshmuttinti Why are we searching for cities, and country, more importantly, are we adding these details into products anywhere.

**Review by rajeshn95 — CHANGES_REQUESTED** [review]

File: database/migrations/20220322155856-create-gm-products-table.js (hunk 1 above)
Comment (maheshmuttinti) [reply database/migrations/20220322155856-create-gm-products-table.js:22]: If the user wants to add the category to a particular product. we need this `category` column while adding product @rajeshn95. Am I right? or does anything here needs changes?

File: app/functions/getGMBidsSearchFilterQueries.js (hunk 2 above)
Comment (maheshmuttinti) [reply app/functions/getGMBidsSearchFilterQueries.js:16]: The user would add country and city while he adding products. That's why I have added the country and cities searching. Can you tell me if this is necessary or not?

Comment (maheshmuttinti) [reply app/functions/getGMBidsSearchFilterQueries.js]: I removed this function since it was wrong implementation. @rajeshn95

**Pull request closed without merging**